sycl/test/basic_tests/buffer/buffer_full_copy.cpp

```diff
@@ -0,0 +1,183 @@
+// RUN: %clangxx %s -o %t1.out -lstdc++ -lOpenCL -lsycl
```

Comment:
Please, remove linking with SYCL and stdc++ libraries.

Reply:
Done